sklearn/ensemble/_forest.py

```diff
@@ -1253,6 +1253,7 @@ class labels (multi-output problem).
     >>> print(clf.predict([[0, 0, 0, 0]]))
     [1]
     """
+
```

Comment:
Could you please avoid changing the style of the code on lines that are not directly related to the topic of the PR?

Such changes can cause unnecessary conflicts in other PRs once merged for little benefit.

Reply:
fix it on https://github.com/scikit-learn/scikit-learn/pull/20159/commits/722f084a1c89aafb624b12f8b6ca060f824e34c0